04-concept-api/references/attribute.yml

```diff
@@ -1,7 +1,7 @@
 methods:
```

Comment:
I think you should tag all the methods only available on `Remote` Concepts clearly, like prefix the title with [Remote Only] in bold. Then group them together at the bottom of each page.

Reply:
iirc - things like `value()` are available on both right?